app/src/main/java/com/android/solvit/provider/ui/profile/ProviderProfile.kt

```diff
@@ -205,6 +208,35 @@ fun ProfileHeader(
           ) {
             Text("Logout")
           }
+
+          if (showLogoutDialog) {
+            AlertDialog(
```

Comment:
Nice addition of the logout confirmation dialog. Consider adding a loading state while the logout is processing to prevent double-clicks and provide better user feedback. You could use a state variable to handle this:
```
var isLoggingOut by remember { mutableStateOf(false) }
```